react/src/lib/components/DeckGLMap/layers/map/mapLayer.ts

```diff
@@ -32,6 +32,7 @@ export type Params = {
     propertiesData: Float32Array;
     isMesh: boolean;
     frame: Frame;
+    enableSmoothShading: boolean;
```

Comment:
The `enable` prefix is not consistent with other component prop naming. Suggest `smoothShading` instead.